test/end-to-end/pages/common/common.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undefined, no-invalid-this */
+
+async function waitForPage(header, headerText) {
```

Comment:
codecept has a helper class we can extend for our helpers :) 
See for example "generalHelper" in https://github.com/hmcts/fprl-ccd-definitions/pull/22/files